src/pages/FarmingPage.js

```diff
@@ -1,19 +1,43 @@
-import React, { useEffect } from "react";
+import React, { useState, useEffect } from "react";
 import "feather-icons";
 
 import { TYPE } from "../Theme";
 import Panel from "../components/Panel";
 import {
-  useLiqudityMiningCampaignData,
+  useLiquidityMiningCampaignData,
 } from "../contexts/PairData";
+
 import { PageWrapper, FullWrapper } from "../components";
 import { RowBetween } from "../components/Row";
 import Search from "../components/Search";
+import DropdownSelect from "../components/DropdownSelect";
 import { useMedia } from "react-use";
 import FarmingList from "../components/FarmingList";
 
 function FarmingPage() {
-  const campaignData = useLiqudityMiningCampaignData();
+
+  const activeCampaigns = useLiquidityMiningCampaignData("active");
+  const expiredCampaigns = useLiquidityMiningCampaignData("expired");
```

Comment:
I think the rest of the code looks fine....Im just not sure if it's best option to have 2 queries like this 🤔  ? Not sure what would alternative be though....